app/Http/Controllers/Api/MeteringPointController.php

```diff
@@ -34,24 +38,19 @@ public function __construct(GetMeteringData $meteringDataService)
      *
      * @return \Illuminate\Http\Response | JsonResponse
      */
-    public function index(string $refresh_token = null)
+    public function index(string|null $refresh_token = null)
```

Comment:
Jeg tror den mere "moderne" måde at vise man tillader `null` er ved at bruge `?` - total smagssag dog 😄 

```php
    public function index(?string $refresh_token = null)
```

Reply:
Arh ja, klart :)
Det har jeg faktisk aldrig gjort før, så det skal jeg lige vænne mig til.
Jeg prøver rundt omkring at skrive lidt anderledes, end jeg plejer, så jeg netop får nye strømninger.
```$awesome = $myValue ?? null;```
Den har jeg f.eks. ikke brugt ret meget før, plejer gerne at skrive `$myValue ? $myValue : null` eller hvis det er vildt: `$myValue ? : null`
Jeg så også, at man kunne noget sejt med $myValue?->property 
Det er lidt en tand for forvirrende for mig lige nu :p